apps/bookmark/test/marionette/install_app_test.js

```diff
@@ -22,8 +22,8 @@ marionette('Bookmark -', function() {
 
   setup(function() {
     system = client.loader.getAppClass('system');
+    system.waitForFullyLoaded();
```

Comment:
I'm not really sure this name change is needed for integration testing, as generally the tests will only care about the system being loaded. I think leaving this as ```waitForStartup``` is probably ok in the context of integration tests. If not though, perhaps we could rip the tests changes out and land them before the big patch lands? It would probably make the review process easier.